.spelling

```diff
@@ -5,3 +5,11 @@ Smallpeice
 SourceBots
 00am
 00pm
+15pm
+45am
+45pm
+30am
+30pm
+40pm
+10pm
+55pm
```

Comment:
Seriously? Is there no way to specify `\d\d(a|p)m` or something? maybe even just ignore all words with numbers in?

Reply:
Not that I know of, but i'll add it to my list to investigate!